recipes/mysql-connector-c/all/test_package/conanfile.py

```diff
@@ -0,0 +1,21 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        bin_path = os.path.join("bin", "test_package")
```

Comment:
This still looks weird to me, can't we simplify this?

https://github.com/bincrafters/templates/blob/c1656604f06cfdaf3a3b45caa69694a11d4ff35a/default/test_package/conanfile.py#L15

Reply:
this is convoluted, but is necessary because of https://github.com/conan-io/conan/issues/5245. I'll simplify it when conan 1.19 is out